bindings/nodejs/__test__/index.spec.mjs

```diff
@@ -15,20 +15,23 @@
 
 import test from 'ava'
 
-import { OperatorFactory } from '../index.js'
+import { Memory } from '../index.js'
 
 test('test memory write & read', async (t) => {
-  let op = OperatorFactory.memory()
+  let builder = new Memory()
+  let op = builder.build()
```

Comment:
Currently, for memory, there are no options. 
In the future, we can use like `builder.bucket("test")` to set the option.